lib/paysafe/account_management_service.rb

```diff
@@ -0,0 +1,225 @@
+module Paysafe
+  class AccountManagementService
+    def initialize client
+      @client = client # PaysafeApiClient
+      @uri = "/accountmanagement/v1" # URI for Account Management API
+    end
+
+    def available
+      request = Request.new(
+      method: Request::GET,
+      uri: "/accountmanagement/monitor",
+      )
+
+      response = @client.process_request request
+      return response.status == "READY"
+    end
+
+    def createMerchantEntity merchantEntity
+      request = Request.new(
+      method: Request::POST,
```

Comment:
I realize we're matching the existing style, but I still feel like I have to mention how awful this indentation pattern is.

Reply:
I looked again and the existing style is inconsistent so I'll do some cleanup.